scripts/apply_weights.py

```diff
@@ -65,7 +81,11 @@
         if line[0] == 'Diseases:':
             diseases = line[1:]
         if line[0] == 'Coefficients:':
-            coef_df = pd.read_table(line[1])
+            coef_df = line[1]
```

Comment:
The way the summary file was constructed (e.g. [here](https://github.com/greenelab/pancancer/blob/master/classifiers/RAS/classifier_summary.txt)) implies that the coefficients for the specific classifier (of which to apply the weights using) was already generated.

I guess I am confused about the specific scenario where the classifier file doesn't exist, but the summary file does.

Reply:
If e.g. I use a job staging submission process on a cluster, the Job that writes `classifier_summary.txt` might do so in a job/node specific directory `/a/b/c/`, which would set the value for `Coefficients` to `/a/b/c/classifier_coefficients.tsv` in the classifier summary file. The written directories and files, can then be staged back out to a persistent directory, maybe `/x/y/z/`.

On a subsequent job, that makes use of the classifier, it may stage files in at `/q/r/s/` (or use `/x/y/z/` directly), so  `/q/r/s/classifier_summary.txt` (loaded by user provided file path) has a `Coefficients` value that points to `/a/b/c/classifier_coefficients.tsv`, but `/a/b/c` does not exist, the file is actually at `/q/r/s/classifier_coefficients.tsv`.